builder/builder.py

```diff
@@ -336,14 +336,16 @@ def publish(course_key: str) -> List[str]:
 
     config = None
     errors = []
+    tmpfiles = list()
```

Comment:
`list()` and `[]` are the same thing. It is just a little curious that you use another form than the line above.